runtime_test.go

```diff
@@ -1036,7 +1036,7 @@ func TestToValueNil(t *testing.T) {
 	}
 
 	var ar []interface{}
-	if v := vm.ToValue(ar); !IsNull(v) {
+	if v := vm.ToValue(ar); IsNull(v) {
```

Comment:
[q] I am curious about this sudden testing inversion. Was ToValue incorrect in the older goja versions?

Reply:
Ah I think I tracked this down, this looks good, when calling `ToValue` they removed a check:

```go
        // ...
	case []interface{}:
                // This has been removed with this PR
		// if i == nil {
		// 	return _null
		// }
		return r.newObjectGoSlice(&i).val
       //...
```

and I assume they expect an empty array to be returned as a value rather than `null` so the testing inversion makes sense.